src/applications/personalization/profile/components/direct-deposit/DirectDepositV2.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
This component is very much WIP! It is currently gated in all environments. The feature flag is not turned on in staging so this will not be used anywhere except locally.